etc/docker/Dockerfile.nvcr-triton-containers

```diff
@@ -1,4 +1,4 @@
-ARG TRITON_VERSION=24.09
+ARG TRITON_VERSION=25.02
```

Comment:
25.03 is the latest release, but 25.02 is the last one with builtin TF support, so we will be stuck here until we do the work to build the TF backend as part of the container build for non-JP5 platforms.

Reply:
Can you put a comment saying as much in the dockerfile, too?

Reply:
Done